src/run-context/runContext.test.ts

```diff
@@ -6,11 +6,11 @@ describe('RunContext', () => {
   describe('incrementPc', () => {
     test('should successfully increment pc', () => {
       const ctx = new RunContext();
-      const instructionSize = UnsignedInteger.toUint64(2n);
-      const result = ctx.incrementPc(instructionSize);
+      const instructionSize = UnsignedInteger.toUint32(2).unwrap();
```

Comment:
how come we can call unwrap() here?

Reply:
don't think we can, I don't know why this passed